bindata/etcd/etcd-member-remove.sh

```diff
@@ -17,21 +17,40 @@ if [ "$1" == "" ]; then
     usage
 fi
 
-ETCD_NAME=$1
-ASSET_DIR=/home/core/assets
-ASSET_DIR_TMP="$ASSET_DIR/tmp"
-ETCDCTL=$ASSET_DIR/bin/etcdctl
-ETCD_DATA_DIR=/var/lib/etcd
-CONFIG_FILE_DIR=/etc/kubernetes
+source etcd.env
+
+# download etcdctl from upstream release assets
+function dl_etcdctl() {
+  ETCDCTL_BIN="${ASSET_DIR}/bin/etcdctl"
+  local etcdimg="registry.svc.ci.openshift.org/ci-op-l7y8kffx/stable@sha256:c576980bdce812e5d000e6f96fc238d75688da01f767cdaed6cbf04c40c600fa"
```

Comment:
This is wrong. The image should come from `local etcdimg="{{.Images.etcdKey}}"`

Reply:
> This is wrong. The image should come from `local etcdimg="{{.Images.etcdKey}}"`

ok, you'll need to do that in the golang code.